nncf/quantization/algorithms/fast_bias_correction/backend.py

```diff
@@ -170,11 +176,43 @@ def process_model_output(raw_data: OutputType, output_name: str) -> NNCFTensor:
 
     @staticmethod
     @abstractmethod
-    def is_node_with_bias(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+    def is_node_with_bias(node: NNCFNode, nncf_graph: NNCFGraph, model: TModel) -> bool:
         """
         Checks whether the node has a bias or not.
 
         :param node: NNCFNode with the attributes.
         :param nncf_graph: NNCFGraph that contains node.
+        :param model: Backend-specific model.
         :return: Boolean indicating whether the node has a bias or not.
         """
+
+    @staticmethod
+    @abstractmethod
+    def get_bias_shift_magnitude(current_bias_value: TTensor, updated_bias_value: TTensor) -> float:
+        """
+        Calculates bias shift magnitude based on the current and updated values.
+
+        :param current_bias_value: The original bias value.
+        :param updated_bias_value: The updated bias value.
+        :return: Magnitude between original and updated bias values.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def post_process_output_data(data: List[TTensor]) -> TTensor:
+        """
+        Convert data to backend specific type.
+
+        :param data: List of data.
+        :return TTensor: Converted data.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def reshape_tensor(data: TTensor, new_shape: List[int]) -> TTensor:
```

Comment:
I would rephrase the method's naming cause it's too general and seems to be in common part.

Reply:
It's general name for general function. Why is it problem?